service/client.go

```diff
@@ -38,6 +40,52 @@ type Client struct {
 	wg  sync.WaitGroup
 }
 
+type ClientConfig struct {
```

Comment:
This code was simply moved from `config.go` since it made more sense to keep it closer to the rtcd Client implementation.